src/components/YourCompany/NameTranslations/ListNameTranslation.vue

```diff
@@ -20,9 +20,12 @@
         <v-chip v-if="translation.action === ActionTypes.ADDED"
           x-small label color="#1669BB" text-color="white">ADDED</v-chip>
         <v-chip v-if="translation.action === ActionTypes.EDITED"
-          x-small label color="#1669BB" text-color="white">CORRECTED</v-chip>
+          x-small label color="#1669BB" text-color="white">
```

Comment:
Please use Vuetify theme color name here (and replace any others you find).

See https://github.com/bcgov/business-edit-ui/blob/800fbf3cb3049cfbb739ca5e83570e1762e57313/src/main.ts#L71

Reply:
Done